src/interaction.ts

```diff
@@ -29,6 +30,18 @@ export default class Interaction {
 		});
 	}
 
+	// TODO: Type? Should return an object where keys = #options keys, and value = ApplicationCommandInteractionDataOption
+	get options() {
+		return new Proxy(
+			{},
+			{
+				get: (target, property): OptionType | null => {
+					return this.#options.get(property.toString())?.value ?? null;
+				},
+			}
+		);
+	}
```

Comment:
We should look into moving this away from a proxy in the future as they cause more headaches than good imho